template-only-bin/update-template.sh

```diff
@@ -34,16 +34,16 @@ INCLUDE_PATHS=" \
   .grype.yml \
   .hadolint.yaml \
   .trivyignore"
-git diff $CURRENT_VERSION $TARGET_VERSION -- $INCLUDE_PATHS > patch
+git diff "$CURRENT_VERSION" "$TARGET_VERSION" -- "$INCLUDE_PATHS" > 'patch'
```

Comment:
Are the quotes needed around `patch`?
```suggestion
git diff "$CURRENT_VERSION" "$TARGET_VERSION" -- "$INCLUDE_PATHS" > patch
```

Reply:
yeah, apparently since there's an actual linux command called `patch`, shellcheck thinks i might have accidentally meant to do `git diff | patch` instead of `git diff > patch`. i could also use a different filename for the patch if you prefer.

Reply:
ok renamed